IBAnimatable/BorderSide.swift

```diff
@@ -22,6 +26,10 @@ public struct BorderSides: OptionSet {
   public static let right = BorderSides(rawValue: 1 << 1)
   public static let bottom = BorderSides(rawValue: 1 << 2)
   public static let left = BorderSides(rawValue: 1 << 3)
+  public static let topOut = BorderSides(rawValue: 1 << 4)
+  public static let rightOut = BorderSides(rawValue: 1 << 5)
+  public static let bottomOut = BorderSides(rawValue: 1 << 6)
+  public static let leftOut = BorderSides(rawValue: 1 << 7)
   
   public static let AllSides: BorderSides = [.top, .right, .bottom, .left]
```

Comment:
Yes, if someone wants all, then can put it all together manually. I am OK with that.